src/java/org/apache/cassandra/tools/nodetool/Verify.java

```diff
@@ -89,6 +99,9 @@ public void execute(NodeProbe probe)
             System.exit(1);
         }
 
+        if (onlySai && includeSai)
+            throw new IllegalArgumentException("Cannot specify both --sai-only and --include-sai");
```

Comment:
why is this an issue?  We do `options.onlySai || options.includeSai`

I would think `--sai-only` would implicitly set `includeSai = true`

Reply:
im ok if we want to keep this, but similar APIs have added flags to change the defaults (over time as they get used); maybe we *should* validate SAI by default?  If we make that a flag than `onlySai` becomes unsafe